src/grammar.coffee

```diff
@@ -351,7 +351,7 @@ grammar =
   # A splat that occurs outside of a parameter list.
   Splat: [
     o 'Expression ...',                         -> new Splat $1
-    o '... Expression',                         -> new Splat $2
+    o '... Expression',                         -> new Splat $2, {postfix: no}
```

Comment:
Preserve whether the splat dots were prefix or postfix

There will be similar updates in the grammar for param and object rest splats as we get to them